webapi/src/main/scala/org/knora/webapi/routing/Authenticator.scala

```diff
@@ -855,6 +814,22 @@ object Authenticator extends InstrumentationSupport {
       // _ = log.debug(s"getUserByIdentifier - user: $user")
     } yield user
   }
+
+  /**
+   * Calculates the cookie name, where the external host and port are encoded as a base32 string
+   * to make the name of the cookie unique between environments.
+   *
+   * The default padding needs to be changed from '=' to '9' because '=' is not allowed inside the cookie!!!
+   * This also needs to be changed in all the places that base32 is used to calculate the cookie name, e.g., sipi.
+   *
+   * @param settings
+   */
+  def calculateCookieName(settings: KnoraSettingsImpl): String = {
+    //
```

Comment:
Shouldn't it be private method?
Also `//` to remove.

Reply:
no, this method is used in some places other than in this file.